test/humancellatlas/data/metadata/helpers/schema_examples.py

```diff
@@ -0,0 +1,50 @@
+from io import BytesIO
+import tarfile
+import urllib.request
+import uuid
+import json
+
+from dcplib.checksumming_io import ChecksummingSink
+
+
+def download_example_bundle(repo, branch, path='/'):
+    manifest = []
+    metadata_files = {}
+    if path.startswith('/') or not path.endswith('/'):
+        raise ValueError(path)
+    with urllib.request.urlopen(f"https://github.com/{repo}/tarball/{branch}") as f:
+        # We need to buffer the response because `tarfile` requires a seekable file object when the tarball is
+        # compressed whereas the file object returned by `urlopen` is not seekable.
+        buf = BytesIO(f.read())
+        with tarfile.open(fileobj=buf) as tf:
+            for member in tf.getmembers():
+                # Member names always start with a synthetic root dir containing the repo name and commit hash
+                member_path = member.name.partition('/')[2]
+                if member_path.startswith(path):
+                    member_path = member_path[len(path):]
+                    if member_path.endswith('.json'):
+                        with tf.extractfile(member) as tfm:
+                            contents = tfm.read()
+                        json_contents = json.loads(contents)
+                        metadata_files[member_path] = json_contents
+                        sink = ChecksummingSink(64 * 2 ** 20)
+                        sink.write(contents)
+                        manifest.append({**sink.get_checksums(),
+                                         'content-type': 'application/json',
```

Comment:
Would it matter to add `'content-type': 'application/json; dcp-type="metadata/xxx"'` here?

Reply:
Yes, good catch.